### DISCUSSION TITLE
Model running difficulty

### DISCUSSION THREAD

**Discussion by saxenasatyam** [magpiemodel/magpie#375] [Q&A] [answered]
Hi,

I am running a model, but in the output folder, I m not getting files developed while running of model. Getting only two files in the output folder and some are coming in the main folder. The main folder means where we run CMD for MAgPIE run.  [](url)
![image](https://user-images.githubusercontent.com/71261663/153359569-66a25ad1-dbca-4cb5-abb2-bd0f4cf7c823.png)
![image](https://user-images.githubusercontent.com/71261663/153359626-5174f96f-c726-419a-8379-e390db708572.png)

**Comment by tscheypidi** [comment]
Hi Satyam,

from the files in these folders it is not possible to say what is going wrong. What log messages do you get when starting the run?

**Comment by saxenasatyam** [comment]
Is this message you are asking for.
![image](https://user-images.githubusercontent.com/71261663/153383975-241a2383-f095-4303-a917-798cb423122a.png)

**Reply by tscheypidi** [reply]
yes, but is this the complete output? Normally, there should be additional log messages after `-> rev4.65_h12.validation.tgz`. What happens after it?

In addition, it would be easier for debugging if you could copy the log messages as text and not as image.

**Comment by saxenasatyam** [comment]
Actually model is still running. So I thought to send you starting lines as an image. Once it will be finished, I'll send you.

**Comment by saxenasatyam** [comment]
Hi Jan, 

My code is running into infinity. It's not getting stopped. Still I am sending you my run into txt formate. Please find attached.[](url
[Test Run.txt](https://github.com/magpiemodel/magpie/files/8040845/Test.Run.txt)
)

**Reply by tscheypidi** [reply]
looks ok to me, you probably just have to wait a bit longer: what I saw is that the calibration file you wanted to download was not available, which is why the model now tries to recalibrate own its own and this recalibration can take a while

**Reply by abhimishr** [reply]
There are some additional ways to diagnose the reasons behind longer times for model runs (compared to a default run) on top of what Jan already mentioned in this case (i.e., recalibration).
1. Check if the config is drastically different compared to a known default run.
2. Check is the `full.gms` differs from a known default run (in case there are additional equations added by the user slowing down the model)

**Comment by saxenasatyam** [comment]
Hi, Abhi thanks for the suggestion, 

My run won't be able to generate full.gms file. In my output folder, I am getting only two files (attached picture) and some files are generated in the main (attached picture)
![image](https://user-images.githubusercontent.com/71261663/154002386-ad17411a-2b3d-4f20-b9b2-bb5e76c797af.png)   
![image](https://user-images.githubusercontent.com/71261663/154004841-8f8d5edb-990c-4382-8056-61a11c7b1c2c.png)

Thanks in advance!

**Reply by abhimishr** [reply]
The reason is that you have a `.lock` file which does not allow any additional runs to be created. Perhaps a run you started and canceled immediately? 
I can also see that you even have some runs "within" the model folder and with the `225*` folders looks like either a run is still running there or it was stopped prematurely.

Easiest would be to do the following: 

1. Close all the command prompts currently open. 
2. Delete the `.lock` file
3. Delete all `*.gdx` files from your model folder (`livestockind` in your case) - I think you do not need them as I do not see any collated `fulldata.gdx` here
4. Delete all model output folders in your `./output` folder which have just two files (does not matter the name - one would be `runstatistics.rda` and another RWorkspace file - similar to what you see in first image. 
4. Restart your local machine (or kill any background `gams` instances if you prefer that)
5. Start a fresh run - ideally with a title which does not match any of the folders in you `./output` directory

**Comment by saxenasatyam** [comment, marked as answer]
Thanks for the direction Abhi. 
I have made specific changes, now stuck with a new problem. 
[failed model.txt](https://github.com/magpiemodel/magpie/files/8087207/failed.model.txt)

I am not able to download complete input data. Some input files are FAILED to download.   
Can you please guide me, why some files are showing failed and how to resolve it? 

Thanks in advance!

**Reply by tscheypidi** [reply]
the data was missing in the public data repository. Now it is there and the download should be possible

**Comment by saxenasatyam** [comment]
Thank you so much. Perfectly working.
